docs/changelog-fragments/1.feature.rst

```diff
@@ -0,0 +1,2 @@
+Add cython extension for libssh client
```

Comment:
To make a good changelog, the sentence style shouldn't be imperative. They should be written in the past tense.

Reply:
The original PR had links to the PR/issue which seems to be removed. imo the changelog should link to PR/issues for which the change was done.

Reply:
The links are made from the numbers in fragment names. Do you mean this?

Reply:
Ah, I missed that link. Ignore my earlier comment.

Reply:
Fixed